### PR TITLE
Us044 View RU details

### PR DESCRIPTION
**Pull request by andrewmil**
Added test scenarios for us044

[Trello](https://trello.com/c/zHV82xsH/77-us044-int-view-ru-details-med)
[Confluence](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/248643696/US044+INT+Display+the+Reporting+Unit+Details+Page+Medium)

### CODE REVIEW

**Comment by andrewmil** [comment]
[US044: INT View RU Details [Med]](https://trello.com/c/zHV82xsH/77-us044-int-view-ru-details-med)

File: acceptance_tests/features/pages/reporting_unit.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from acceptance_tests import browser
+from config import Config
+
+
+def go_to(ru_ref):
+    browser.visit(f"{Config.RESPONSE_OPERATIONS_UI}/reporting-units/{ru_ref}")
```
Comment (rhiGriff) [inline acceptance_tests/features/pages/reporting_unit.py]: Can we keep these consistent across the repo please?

Comment (andrewmil) [reply acceptance_tests/features/pages/reporting_unit.py]: I had a look into it and f-strings are actually a faster method of making strings like this and they're more readable. 
I'm happy to change it back but in the longer run I think this is the better way to go

Comment (benjefferies) [reply acceptance_tests/features/pages/reporting_unit.py]: I'd agree that f-strings are actually more readable and I think as a team we've adopted f-strings in the majority of other repositories. Two ways of changing it are raise a tech debt to change it all or change it as we go

File: acceptance_tests/features/view_reporting_unit_details.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Feature: Display reporting unit page
+  As an internal user
+  I need to to be able to view reporting unit details
+  So that I can undertake a number of potential actions related to that reporting unit
+
+  Background: Internal user is already signed in
+    Given the internal user is already signed in
+
+  @us044-RUDetailsPresentedToUser_s01
```
Comment (benjefferies) [inline acceptance_tests/features/view_reporting_unit_details.feature]: Have we got an agreed format for these tags? In it's current form it's really ugly. Just the user story or a simple feature description would be more readable

Comment (andrewmil) [reply acceptance_tests/features/view_reporting_unit_details.feature]: I copied the style that was in the `add_a_survey.feature` file. We should agree on a better format though i agree. Is that work for this PR?

File: acceptance_tests/features/view_reporting_unit_details.feature [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Feature: Display reporting unit page
+  As an internal user
+  I need to to be able to view reporting unit details
+  So that I can undertake a number of potential actions related to that reporting unit
+
+  Background: Internal user is already signed in
+    Given the internal user is already signed in
+
+  @us044_s01
```
Comment (gemmai95) [inline acceptance_tests/features/view_reporting_unit_details.feature]: Is this the decided way to go now?

**Comment by benjefferies** [comment]
@andrewmil needs a rebase

File: acceptance_tests/features/view_reporting_unit_details.feature (hunk 3 above)
Comment (andrewmil) [reply acceptance_tests/features/view_reporting_unit_details.feature]: Not right now but we can come to a decision in the future and align them
Don't think they're getting used anywhere atm

File: acceptance_tests/features/steps/view_reporting_unit_details.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    pass
+
+
+@when('the internal user views the 49900000001 reporting unit page')
+def internal_user_views_the_survey_page(_):
+    reporting_unit.go_to('49900000001')
+
+
+@then('the internal user is displayed the correct reporting unit details')
+def internal_user_views_correct_reporting_unit_details(_):
```
Comment (benjefferies) [inline acceptance_tests/features/steps/view_reporting_unit_details.py]: This was failing for me
```
  @us044_s01
  Scenario: The reporting unit details are presented to the user           # acceptance_tests/features/view_reporting_unit_details.feature:10
    Given the internal user is already signed in                           # acceptance_tests/features/steps/authentication.py:19 0.529s
    Given the reporting unit 49900000001 is in the system                  # acceptance_tests/features/steps/view_reporting_unit_details.py:6 0.000s
    When the internal user views the 49900000001 reporting unit page       # acceptance_tests/features/steps/view_reporting_unit_details.py:11 2.022s
    Then the internal user is displayed the correct reporting unit details # acceptance_tests/features/steps/view_reporting_unit_details.py:16 2.224s
      Traceback (most recent call last):
        File "/Users/jeffeb/.local/share/virtualenvs/ras-integration-tests-EA68moDt/lib/python3.6/site-packages/splinter/element_list.py", line 40, in __getitem__
          return super(ElementList, self).__getitem__(index)
      IndexError: list index out of range
      
      During handling of the above exception, another exception occurred:
      
      Traceback (most recent call last):
        File "/Users/jeffeb/.local/share/virtualenvs/ras-integration-tests-EA68moDt/lib/python3.6/site-packages/splinter/element_list.py", line 72, in __getattr__
          return getattr(self.first, name)
        File "/Users/jeffeb/.local/share/virtualenvs/ras-integration-tests-EA68moDt/lib/python3.6/site-packages/splinter/element_list.py", line 53, in first
          return self[0]
        File "/Users/jeffeb/.local/share/virtualenvs/ras-integration-tests-EA68moDt/lib/python3.6/site-packages/splinter/element_list.py", line 44, in __getitem__
          self.find_by, self.query))
      splinter.exceptions.ElementDoesNotExist: no elements could be found with id "RU_NAME"
      
      During handling of the above exception, another exception occurred:
      
      Traceback (most recent call last):
        File "/Users/jeffeb/.local/share/virtualenvs/ras-integration-tests-EA68moDt/lib/python3.6/site-packages/behave/model.py", line 1456, in run
          match.run(runner.context)
        File "/Users/jeffeb/.local/share/virtualenvs/ras-integration-tests-EA68moDt/lib/python3.6/site-packages/behave/model.py", line 1903, in run
          self.func(context, *args, **kwargs)
        File "acceptance_tests/features/steps/view_reporting_unit_details.py", line 18, in internal_user_views_correct_reporting_unit_details
          ru_details = reporting_unit.get_ru_details()
        File "/Users/jeffeb/source/github/onsdigital/ras-integration-tests/acceptance_tests/features/pages/reporting_unit.py", line 11, in get_ru_details
          "name": browser.find_by_id('RU_NAME').text,
        File "/Users/jeffeb/.local/share/virtualenvs/ras-integration-tests-EA68moDt/lib/python3.6/site-packages/splinter/element_list.py", line 75, in __getattr__
          self.__class__.__name__, name))
      AttributeError: 'ElementList' object has no attribute 'text'
```

**Pull request merged**